content/pages/privacy-policy.md

```diff
@@ -0,0 +1,105 @@
+title: Mixxx Privacy Policy
+slug: privacy-policy
+
+## Introduction
+
+Mixxx is a non-profit project, managed by volunteers.
+
+There is no business-model around Mixxx, and the only revenue we receive is through donations.
+
+For privacy policies and conformance to GDPR, a few distinctions will be done between:
+
+- Mixxx application
+- Main Mixxx website
+- Community Mixxx websites
+
+For each of the previous software/websites, we'll detail the different privacy policies.
+
+## Mixxx application
+
+The Mixxx application doesn't use any user account, and does not collect any user data, when running.
+
+Mixxx does not collect any data, nor any telemetry, when Mixxx is being run.
+However, some limited personal information could be collected or shared in 3 precise cases:
+
+- during the crash report process,
+- during updates checking,
+- for metadata retrieval.
+
+## Mixxx crash reporting
+
+The Mixxx application itself neither collects crash reports nor shares them.
+
+Crash reports are handled by the Stores/Vendors of each platforms.
+Please refer to those platforms for information about the privacy policies.
+Depending on the platform, Mixxx Core Developers have access to these reports, which usually contain:
+
+- Operating System version
+- CPU and GPU
+- Symbolic StackTrace of the process
+- List of DLLs loaded by the process
+
+None of these data is stored on Mixxx servers.
+
+## Mixxx update mechanism
+
+The Mixxx application itself doesn't provide an update mechanism.
+
+Updates are handled by the Stores of each platforms, where the user has downloaded Mixxx from.
+Please refer to those platforms for information about the privacy policies.
+
+## Mixxx metadata requests
+
+Mixxx can automatically search online for metadata associated to your media, to 3rd parties.
+Those requests can contain information about the media, including artist, title, album, etc...
+This metadata mechanism can be disabled at every moment, in Mixxx preferences.
+
+The main 3rd parties that can be contacted by Mixxx, automatically is:
+
+ - Mixxx MusicBrainz instance.
+
+Please refer to those services for their own privacy policy.
+Mixxx does not have access to any of those requests and data, except in the MusicBrainz case, detailed below, in this document.
+Mixxx mobile
+
+## Main Mixxx website
+
+The main website of Mixxx, aka www.mixxx.org, does not create or use any user account.
+There are no registration forms, or any similar mechanism.
+Therefore, no personal information is stored at any moment, about the user, by Mixxx.
+
+## Community Mixxx websites
+
+The community websites from Mixxx are related to development and community, and are the following:
+
+- GitHub
+- Discourse Forum
+- Zulip Chat
+
+Each of those websites have different users system, but work in a similar way.
+
+No user information is processed in any way other than being able to log in and have a profile.
+
+On GitHub no user information is shared to any 3rd party.

```

Comment:
Really? Not sure about that. Shouldn't we jus link to their privacy policy instead?